examples/src/indexes/vector_stores/vectara.ts

```diff
@@ -58,3 +58,53 @@ console.log(JSON.stringify(resultsWithScore, null, 2));
 //     0.38169062
 //   ]
 // ]
+
+const resultsWithSummary = await store.similaritySearchWithSummary(
+  "What were the women talking about?",
+  1,
+  {
+    lambda: 0.025,
+    summary: [
+      {
+        summarizerPromptName: "vectara-summary-ext-v1.2.0",
+        maxSummarizedResults: 3,
+        responseLang: "ita",
+      },
+    ],
+  }
+);
+
+console.log(JSON.stringify(resultsWithSummary, null, 2));
+// {
+//  "documents": [
+//   [
+//     {
+//       "pageContent": "In the room the women come and go talking of Michelangelo",
+//       "metadata": [
+//         {
+//           "name": "lang",
+//           "value": "eng"
+//         },
+//         {
+//           "name": "offset",
+//           "value": "0"
+//         },
+//         {
+//           "name": "len",
+//           "value": "57"
+//         }
+//       ]
+//     },
+//     0.38169062
+//   ]
+// ],
+// "summary": [
+//   {
+//      "text": "Nella stanza le donne vanno e vengono parlando di Michelangelo. Inoltre, sembra che stiano valutando se mangiare una pesca.",
+//      "lang": "ita",
+//      "prompt": "",
+//      "status": [],
+//      "futureId": 2
+//   }
+//  ]
+// }
```

Comment:
nit: remove? 
```suggestion

```

Reply:
This is an example file (note the directory path). I'm following the existing pattern of writing JavaScript code with comments purposely highlighting the supposed console log output.

Any ideas for better formatting?

Reply:
We usually make it a comment block to allow for easier copy/paste but am not super picky:
/*
  {
    ...
  }
*/

Reply:
Updated!